modules/administration/pages/security-hardening.adoc

```diff
@@ -0,0 +1,54 @@
+[[security-hardening]]
+= Security Hardening
+
+While no implementation is ever completely secure from bad actors, there are some things you can do within {productname} to improve your security.
+
+This section contains a number of suggestions for hardening your security posture.
+The primary focus is to reduce the likelihood that a bad actor could access Salt clients in the case of a breach of the {productname} Server.
+In this case, we assume that the Salt master is running on the {productname} Server.
+
+To begin, there are some built-in mechanisms that make the {productname} Server less vulnerable to bad actors.
```

Comment:
the next sentence is unrelated to this sentence (it's not a built-in mechanism and it would be the same on any other server system).  What war the built-in mechanisms?

Reply:
That was me trying to organise the content better, failing, and then forgetting to remove the old sentence :confused: